README.md

```diff
@@ -79,6 +81,8 @@ the names of resources _should_ match what's in this package.
 ### Billing
 | API Resource                                                                                 | Method                                          | Example                                                                                                                                                                                            |
 |----------------------------------------------------------------------------------------------|-------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [Credit Balance Transactions](https://stripe.com/docs/api/credit-balance-transaction)        | `stripeBillingCreditBalanceTransactionId()`     | `cbtxn_37b3DSyi5lc9uck2sp1hs0on`                                                                                                                                                                  |
```

Comment:
This link goes to a 404 page. I think it should be https://docs.stripe.com/api/billing/credit-balance-transaction